__tests__/bikes.test.js

```diff
@@ -0,0 +1,55 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+const Bike = require('../lib/models/Bike');
+
+describe('bikes routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  afterAll(() => {
+    pool.end();
+  });
+
+  it('creates a bike', async () => {
+    const expected = {
+      brand: 'Specialized',
+      model: 'S-Works Tarmac SL7',
+      components: 'Shimano Di2',
+    };
+    const res = await request(app).post('/api/v1/bikes').send(expected);
+    expect(res.body).toEqual({ id: expect.any(String), ...expected });
+  });
+
+  it('gets all bikes', async () => {
+    const expected = await Bike.getAll();
```

Comment:
We should probably add a bike here and expect that bike in addition to the one already added in the `setup.sql` file for a more robust test. Currently our expected, and what gets returned from this route will be the same thing. So if the `getAll` function was broken this test would still pass as currently writte